### PR TITLE
Add points slider snippet for Smile UI

### PR DESCRIPTION
**Pull request by ClaytonPassmore**
* Hasn't been tested yet since Smile UI isn't running on BigCommerce at this point

Pre-merge checklist:
* [x] Test 🎉

### CODE REVIEW

File: bigcommerce-points-slider/smile-points-slider.html [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+    /**
+     * Create a name that will exist and update under the points slider.
+     * Developers should update this to reflect their reward value and
+     * currency.
+     */
+    var generateFulfilledRewardNameFromValue = function(value) {
+      return currencySymbol + value + ' off';
+    };
+
+    // TODO: Add doc link for points product object.
```
Comment (robinsharma) [inline bigcommerce-points-slider/smile-points-slider.html]: Fix todo (either remove or add the link)

File: bigcommerce-points-slider/smile-points-slider.html [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+          return;
+        }
+
+        var $pointsSlider = $('.smile-points-slider');
+        var pointsSliderData = $pointsSlider.data();
+        var pointsProductId = pointsSliderData.pointsProductId;
+
+        smile.fetchPointsProduct(pointsProductId).then(function(pointsProduct) {
+          if (pointsProduct.exchange_type !== 'variable') {
+            // TODO: Add doc link
```
Comment (robinsharma) [inline bigcommerce-points-slider/smile-points-slider.html]: Same here

**Comment by ClaytonPassmore** [comment]
@robinsharma reason I didn't add was because the domain for new docs is `docs-next.smile.io`, so I figured I would just wait until we move the docs over to `docs.smile.io`. What are your thoughts?

**Comment by ClaytonPassmore** [comment]
@robinsharma Going to merge up and then add doc links to all new Smile snippets in a separate PR.

**Pull request merged**